app/uploaders/achievement_badge_uploader.rb

```diff
@@ -0,0 +1,13 @@
+# Image uploader for Achievement Badge
+# This class contains achievement badge specific settings for the image uploader.
+class AchievementBadgeUploader < ImageUploader
```

Comment:
Since there are only two image uploaders for now (User Icon and Achievement Badge), that is fine. If we do have more (currently can only think of course logo), we will need to have separate uploaders if they require slightly different configurations. 

I'll put it in the helper then?